lib/puppet/pops/issues.rb

```diff
@@ -751,8 +750,11 @@ def self.hard_issue(issue_code, *args, &block)
   end
 
   NUMERIC_OVERFLOW = hard_issue :NUMERIC_OVERFLOW, :value do
-    range_end = value > 0 ? _('max') : _('min')
-    _("#{label.a_an_uc(semantic)} resulted in a value outside of Puppet Integer #{range_end} range, got '#{"%#+x" % value}'")
+    if value > 0
```

Comment:
In order to avoid changing the content of error messages that people may be relying on, can you use "max" and "min" here instead of "maximum" and "minimum"?